docs/UserGuide.md

```diff
@@ -336,6 +336,17 @@ Examples:
   `clear`
   `undo` (reverses the `clear` command)
   `undo` (reverses the `cancel 1` command)
+  
+Step by step illustration:
+1. Initial state:
+![UndoCommandInitialState](images/UndoRedoInitialState_UG.png)
```

Comment:
Looks great! Just wondering if it'll make it look even better to add a <br> before and after the pics for the text to look better? Lemme know whatcha think :)